lib/chai/core/assertions.js

```diff
@@ -537,10 +537,11 @@ module.exports = function (chai, _) {
         itemsCount = val.length;
         break;
       case 'function':
-        throw new TypeError('.empty() was passed a function');
+        var name = val.name ? ' ' + val.name : '';
+        throw new TypeError('.empty was passed a function' + name);
       default:
         if (val !== Object(val)) {
-          throw new TypeError('.empty() was passed non-string primitive');
+          throw new TypeError('.empty was passed non-string primitive ' + String(val));
```

Comment:
Okay, calm down everyone:

THUMBS UP IF YOU VOTE FOR `TYPE` AND HEART IF YOU VOTE FOR `INSPECT`.
If we three vote there's no way this is going to be a tie.